### PR TITLE
tests for task1

### PR DESCRIPTION
**Pull request by jimmykimani**

### CODE REVIEW

File: tests/test_dojo.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_create_multiple_offices(self):
         self.dojo.create_room('Blue', 'office')
         self.dojo.create_room('Black', 'office')
         self.dojo.create_room('Brown', 'office')
-        self.assertEqual(len(self.dojo.all_rooms), 3, 'rooms added successfully')
+        self.assertEqual(len(self.dojo.all_rooms), 3,)
```
Comment (Gchesta) [inline tests/test_dojo.py]: What is this testing? The addition of a room or of a person?

File: tests/test_dojo.py [hunk 2]
```diff
@@ -64,5 +64,33 @@ def test_staff_is_added_sucessfully(self):
...
         self.assertEqual(len(self.dojo.all_rooms), 1, 'staff sucessfully added')
+
+    def test_print_room(self):
+        '''test that members of a room are printed'''
+        self.dojo.create_room('blue', 'office')
+        self.dojo.create_room('brown', 'livingspace')
+        self.dojo.add_person('sonia', 'fellow')
+        self.dojo.add_person('kobby', 'staff')
+        self.dojo.add_person('bett', 'fellow')
+        self.dojo.print_room('blue')
```
Comment (Gchesta) [inline tests/test_dojo.py]: Nothing to assert here. The code isn't testing anything? How will you know if it has passed or failed the test?

File: tests/test_dojo.py [hunk 3]
```diff
@@ -64,5 +64,33 @@ def test_staff_is_added_sucessfully(self):
...
+
+    def test_print_allocations_filename(self):
+        '''test that allocated people are printed to a file'''
+        self.dojo.create_room('blue', 'office')
+        self.dojo.create_room('brown', 'livingspace')
+        self.dojo.add_person('kobby', 'staff')
+        self.dojo.add_person('bett', 'fellow')
+        self.dojo.add_person('khalid', 'fellow', 'Y')
+        self.dojo.print_allocations('allocations.txt')
```
Comment (Gchesta) [inline tests/test_dojo.py]: Same as above

File: tests/test_dojo.py [hunk 4]
```diff
@@ -64,5 +64,33 @@ def test_staff_is_added_sucessfully(self):
...
+
+
+    def test_print_unallocated_filename(self):
+        '''test that unallocated people are printed to a file'''
+        self.dojo.create_room('blue', 'office')
+        self.dojo.create_room('brown', 'livingspace')
+        self.dojo.add_person('kobby', 'staff')
+        self.dojo.add_person('bett', 'fellow')
+        self.dojo.add_person('khalid', 'fellow', 'Y')
+        self.dojo.print_un_allocated('unallocated.txt')
```
Comment (Gchesta) [inline tests/test_dojo.py]: Same as above

File: tests/test_dojo.py (hunk 2 above)
Comment (jimmykimani) [inline tests/test_dojo.py]: Here its prints what has already been asserted

Comment (GDP254) [inline tests/test_dojo.py]: Hi, I guess you are looking to test the format or structure fo the output. I guess if you create a variable with the expected output you can find a way of comparing them.

File: App/dojo.py [hunk 5]
```diff
@@ -67,50 +80,61 @@ def check_vacant_room(self):
                 elif len(livingspace.members) >= livingspace.capacity:
                     self.vacant_livingspaces.append(livingspace)
                     self.vacant_rooms.append(livingspace)
```
Comment (GDP254) [inline App/dojo.py]: I think you meant to remove from the stores at this point.

File: tests/test_dojo.py (hunk 2 above)
Comment (RyanSept) [inline tests/test_dojo.py]: Yea this test isn't testing anything. The point of a unit test is to test that the unit is doing what it's supposed to be doing. This simply calls print_room but doesn't assert that it's producing anything. I second Okerosi's suggestion, have it return a string or something and compare that to what you expect.

File: tests/test_dojo.py (hunk 3 above)
Comment (RyanSept) [inline tests/test_dojo.py]: Here you could read the allocations.txt file and assert it's been populated with the data you've added.

**Pull request merged**